src/test/javascript/spec/component/quiz-exercise/quiz-exercise.component.spec.ts

```diff
@@ -62,4 +70,211 @@ describe('QuizExercise Management Component', () => {
         expect(service.findForCourse).toHaveBeenCalled();
         expect(comp.quizExercises[0]).toEqual(quizExercise);
     });
+
+    it('Should reset quiz', () => {
+        const headers = new HttpHeaders().append('link', 'link;link');
+        jest.spyOn(service, 'reset').mockReturnValue(
+            of(
+                new HttpResponse({
+                    body: {},
+                    headers,
+                }),
+            ),
+        );
+        jest.spyOn(service, 'findForCourse');
+
+        comp.course = course;
+        comp.ngOnInit();
+        comp.resetQuizExercise(456);
+        expect(service.reset).toHaveBeenCalledWith(456);
+        expect(service.findForCourse).toHaveBeenCalled();
```

Comment:
```suggestion
        expect(service.findForCourse).toHaveBeenCalledTimes(1);
        expect(service.reset).toHaveBeenCalledWith(456);
```

Please also adapt this for other occurrences in this file